egs/librispeech/ASR/chech_version.py

```diff
@@ -0,0 +1,11 @@
+import torch
```

Comment:
Can you add
```
#!/usr/bin/env python3
```
to the very beginning of this script?

(Also, can you make it executable?)
```python3
chmod +x ./check_version.py
```